krux/stats.py

```diff
@@ -40,42 +40,45 @@ def get_stats(
 
     :argument prefix: The string prepended to all stats sent to Statsd.
 
-    :keyword client: The StatsClient class to use. Defaults to 'True' which
-    signals the use of 'kruxstatsd.StatsClient'. 'False' or 'None' will
-    signal the use of :py:class:`stats.DummyStatsClient
-    <krux.stats.DummyStatsClient>`.  Any other value is interpreted as a
-    custom Stats class.
+    :parameter client: The StatsClient class to use. Defaults to 'True' which
+    signals the use of 'statsd.StatsClient'. 'False' or 'None' will
+    signal the use of :py:class:`stats.DummyStatsClient <krux.stats.DummyStatsClient>`.
+    `legacy` signals the use of kruxstatsd.StatsClient. Any other value is invalid.
 
-    :keyword env: The Statsd environment to report the stat in. Defaults to
+    :parameter env: The Statsd environment to report the stat in. Defaults to
     :py:data:`krux.constants.DEFAULT_STATSD_ENV`
 
-    :keyword host: The Statsd host to connect to. Defaults to
+    :parameter host: The Statsd host to connect to. Defaults to
     :py:data:`krux.constants.DEFAULT_STATSD_HOST`
 
-    :keyword env: The Statsd port to connect to. Defaults to
+    :parameter port: The Statsd port to connect to. Defaults to
     :py:data:`krux.constants.DEFAULT_STATSD_PORT`
     """
 
-    stats = {
+    stats_client = {
```

Comment:
An opportunity to de-obfuscate this black magic?

Reply:
Unfortunately, this is the way to do `switch` statements in Python. 😞